src/rdiff_backup/Globals.py

```diff
@@ -54,6 +55,9 @@
     "max": 201,
     "actual": 0
 }
+# Allow overwrite from the environment variable RDIFF_BACKUP_API_VERSION
```

Comment:
Could we consider a try catch block in case the variable contains garbage ?
```
try:
   ...
catch ScannerError:
   print("a nice user friendly message instead of a stack trace")
   exit(?)
```

Reply:
As it's an advanced feature, I wouldn't mind a traceback, especially as I would anyway raise the exception. But well, why not?

Reply:
OK, I checked why not: because the loading happens too early for handling log messages, and I'd want anyway to avoid  adding a circular dependency between Globals and log. So I only made it clearer that's a developer option.